src/components/overlays/NewNotifications/NotificationView/NotificationHeader.tsx

```diff
@@ -7,16 +7,21 @@ export type NotificationHeaderProps = {
   header: NotificationProps['header'];
 } & HTMLAttributes<HTMLElement>;
 
-const Header = tasty(Title, { gridArea: 'header', margin: '0.25x 0 0.5x' });
+const Header = tasty(Title, {
+  as: 'div',
+  preset: 'h6',
+  styles: {
+    gridArea: 'header',
+    '&:not(:empty)': {
```

Comment:
it should be
```
margin: {
  '': false,
  ':not(:empty)': '0.25x 0. 0.5x',
}
```

But I don't know if it works with brackets.

Reply:
It doesn't work